frontend/src/components/Input/MultiInput.tsx

```
@@ -0,0 +1,74 @@
+    );
+  }
+  return <separator.type {...separator.props} width={ICON_WIDTH}/>;
+};
+
+const intersperse = (
+  childElements: ReactNode[],
+  separator?: JSX.Element,
+): ReactNode[] => 
+  childElements.reduce<ReactNode[]>((acc, child, index) => {
+    if (separator && index < childElements.length - 1) {
+      return [...acc, child, <separator.type {...separator.props} key={index} />];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)